docs/HowTo/Get-Started/Run-Docker-Image.md

```diff
@@ -1,108 +1,127 @@
+---
 description: Run Hyperledger Besu using the official docker image
-<!--- END of page meta data -->
+---
 
-# Running Hyperledger Besu from Docker Image
+# Running Hyperledger Besu from a Docker image
 
-A Docker image is provided to run a Besu node in a Docker container. 
+Besu provides a Docker image to run a Besu node in a Docker container.
 
-Use this Docker image to run a single Besu node without installing Besu. 
+Use this Docker image to run a single Besu node without installing Besu.
 
 ## Prerequisites
 
-* [Docker](https://docs.docker.com/install/)   
+* [Docker](https://docs.docker.com/install/)
 
-* MacOS or Linux 
-    
-    !!! important 
-        The Docker image does not run on Windows. 
+* MacOS or Linux
 
-## Default Node for Mainnet
+    !!! important
```

Comment:
I don't think you need this to be indented. The `!!!` can begin at the start of the line without spaces.

Reply:
Done.